packages/optimizer/lib/RuntimeHostHelper.js

```diff
@@ -17,11 +17,12 @@
 
 const {AMP_CACHE_HOST, appendRuntimeVersion} = require('./AmpConstants.js');
 
-function calculateHost({ampUrlPrefix = AMP_CACHE_HOST, ampRuntimeVersion, lts, rtv = false}) {
-  if (lts && ampRuntimeVersion) {
-    throw new Error('lts flag is not compatible with runtime version parameter');
```

Comment:
I think this error is worth keeping, but `if (lts && ampRuntimeVersion)` should be replaced with `if (lts && rtv)`.

Reply:
Good point. Added.